src/Node/HTTP/HTTPS.purs

```diff
@@ -0,0 +1,242 @@
+-- | This module defines low-level bindings to the Node HTTPS module.
+
+module Node.HTTP.HTTPS
```

Comment:
I think I'd prefer either `Node.HTTPS` or `Node.HTTP.Secure`, what do you think?